### DISCUSSION TITLE
use of three and four letter all caps abbreviations instead of codes with a code_type of CSTM-ALL

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#726] [Q&A]
Has anyone else seen strange billing_codes that seem to be categories rather than specific codes e.g. ENT, HHA, HOS1, HOS2, MISC, MHCD, PTH, SKN, STH? 
They are being combined with a billing_code_type of CSTM-ALL.
And the descriptions seem like categories e.g. PTH has a description of "PHYSICAL THERAPY".

Is there any provision in the schema or rule that would allow this?
We just saw this in some United Healthcare files.

**Comment by onwanalytics** [comment]
Yes, I am seeing these values with United Healthcare files. I can't find anywhere that these are acceptable values. I can make some assumptions on what some of these are, but I do not think these should be reported this way. Does anyone have any reference to the CSTM acceptable values?
billing_code
MHCD
DME
AAIR
ALRG
ALT
ALT1
ALT2
ALT3
AMB
AOB
AUD
BLD
CARD
CHM
CRRH
CSTR
DIAG
DXR
ECHO
EDUC
EEG
EKG
EMG
EMR
FOB
HDL
HOLT
HYPE
IMAG
LAB
NUC
OBS
OCC
ONC
PATH
PDL
PERI
PET
PLMR
PLMT
PSTE
PTH
RAD
ROMI
RSP
SLS
STH
THR
ULT
URC
MISC
MAM
CAT
MRI
SLS1
THR1
THR2
THR4
EMR1
EMR2
EMR3
EMR4
EMR5
TRI
THR3
CLN
TRAU
TRMT
